torchvision/prototype/transforms/functional/_utils.py

```diff
@@ -1,14 +1,17 @@
-from typing import Tuple, cast, Union
+from typing import Tuple, Union
 
 import PIL.Image
 import torch
 from torchvision.prototype import features
 from torchvision.transforms import functional as _F
 
+# FIXME
+
 
 def get_image_size(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> Tuple[int, int]:
     if type(image) is torch.Tensor or isinstance(image, PIL.Image.Image):
-        return cast(Tuple[int, int], tuple(_F.get_image_size(image)))
+        width, height = _F.get_image_size(image)
```

Comment:
We should not be calling the high-level kernel here `_F` but instead dispatch separately like everywhere else for consistency reasons. (also because this API you introduce will replace `F`)